back-end/src/dms/models/books.py

```diff
@@ -39,7 +40,7 @@ class Book(Document):
     publication_year = models.PositiveSmallIntegerField(default=current_year)
     publishers = models.ManyToManyField(to=Publisher, blank=True)
     subjects = models.ManyToManyField(to=Subject, blank=True)
-    cover = models.OneToOneField(to=Cover, on_delete=models.CASCADE, null=True)
+    cover = models.OneToOneField(to=Cover, on_delete=models.SET_NULL, null=True)
```

Comment:
It seems to me that this is a bug: if the cover is removed, it is not necessary to delete the book (unlike the content). I fixed it, and now when the cover is removed, the book is not deleted.

Reply:
A bug indeed, thanks.